### DISCUSSION TITLE
New Prop Suggestion In Input Component

### DISCUSSION THREAD

**Discussion by jcho0216** [heroui-inc/heroui#255] [Feature requests]
Hi! I am a frontend developer from S.Korea. I really love your work and I am currently using this library in my personal project :)

while I was using your Input component I found out that there was no ***Input type="file"*** prop implemented. 
I think this feature is going to be very useful to my further projects

something like this
https://ant.design/components/upload/

thx

**Comment by mariusflorescu** [comment]
What I'm suggesting is another component rather than polluting the existing input component.
PS: This could be moved to the discussions section.

**Comment by JanKrb** [comment]
Anything new here?

**Comment by wingkwong** [comment]
`input type ="file"` is available. see https://storybook.heroui.com/?path=/story/components-input--input-types&globals=locale:bg-BG. For file upload, let's track in https://github.com/heroui-inc/heroui/discussions/1814
